components/dropdown/dropdown-content.js

```diff
@@ -1,23 +1,41 @@
 import { DropdownContentMixin } from './dropdown-content-mixin.js';
 import { dropdownContentStyles } from './dropdown-content-styles.js';
+import { DropdownPopoverMixin } from './dropdown-popover-mixin.js';
 import { LitElement } from 'lit';
 
-/**
- * A generic container for dropdown content.  It provides behavior such as sizing,  positioning, and managing focus gain/loss.
- * @slot - Anything inside of "d2l-dropdown-content" that isn't in the "header" or "footer" slots appears as regular content
- * @slot header - Sticky container at the top of the dropdown
- * @slot footer - Sticky container at the bottom of the dropdown
- * @fires d2l-dropdown-open - Dispatched when the dropdown is opened
- */
-class DropdownContent extends DropdownContentMixin(LitElement) {
-
-	static get styles() {
-		return dropdownContentStyles;
-	}
+const usePopoverMixin = window.D2L?.LP?.Web?.UI?.Flags.Flag('GAUD-7472-dropdown-popover', false);
+if (usePopoverMixin) {
+
+	/**
+	 * A generic container for dropdown content. It provides behavior such as sizing,  positioning, and managing focus gain/loss.
+	 * @slot - Anything inside of "d2l-dropdown-content" that isn't in the "header" or "footer" slots appears as regular content
+	 * @slot header - Sticky container at the top of the dropdown
+	 * @slot footer - Sticky container at the bottom of the dropdown
+	 * @fires d2l-dropdown-open - Dispatched when the dropdown is opened
+	 */
+	class DropdownContent extends DropdownPopoverMixin(LitElement) { }
```

Comment:
Since this is essentially defining the component twice, I wonder what's going to happen with the Daylight site stuff?

Reply:
Yeah I was wondering that too. That's something I have on my list to look into. I'd really like to keep a clean separation of the the two versions.

Reply:
So interestingly, it works fine. 😓 

I've hidden the PopoverMixin events since we don't want them included in the dropdown documentation. i.e. since PopoverMixin is internal to dropdown, and dropdown already exposes events for consumers.